python-checks/src/test/resources/checks/nonCallableCalled.py

```diff
@@ -0,0 +1,113 @@
+class MyNonCallable: ...
+
+class MyCallable:
+    def __call__(self):
+        print("called")
+
+def func(): ...
+
+def call_noncallable():
+    myvar = MyNonCallable()
+    myvar()  # Noncompliant {{Fix this call; "myvar" is not callable.}}
+#   ^^^^^
+
+    none_var = None
+    none_var()  # Noncompliant
+
+    int_var = 42
+    int_var()  # Noncompliant
+
+    list_var = []
+    list_var()  # Noncompliant
+
+    tuple_var = ()
+    tuple_var()  # Noncompliant
+
+    dict_var = {}
+    dict_var()  # Noncompliant
+
+    set_var = set()
+    set_var()  # FN (set has unresolved type hierarchy)
```

Comment:
maybe we can hardcode it in the rule?

Reply:
Done in aaaac0c3236b8c3d45c1c19f26873e584d0a457d. That said, I assume there will be many other types in that situation.